media-libs/opencollada/opencollada-9999.ebuild

```diff
@@ -1,92 +1,78 @@
-# Copyright 1999-2015 Gentoo Foundation
+# Copyright 1999-2016 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 # $Id$
 
-EAPI=5
+EAPI=6
 
-if [[ ${PV} == *9999* ]] ; then
-	SCM_ECLASS="git-r3"
+if [[ ${PV} == "9999" ]] ; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/KhronosGroup/OpenCOLLADA.git"
 else
-	SCM_ECLASS="vcs-snapshot"
+	SRC_URI="https://github.com/KhronosGroup/OpenCOLLADA/archive/v${PV}.tar.gz
+		-> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~ppc64 ~x86"
+	S="${WORKDIR}"/OpenCOLLADA-${PV}
```

Comment:
I know I triggered this, but I'm not too fond of having it all end up on top of the ebuild. As an example how it can be done more 'inline', see: https://github.com/gentoo/gentoo/blob/master/media-libs/libvisio/libvisio-9999.ebuild